PROTO_tests/tests/dtypes_test.py

```diff
@@ -0,0 +1,168 @@
+import numpy as np
+import pytest
+
+import arkouda as ak
+from arkouda import dtypes
+
+"""
+DtypesTest encapsulates arkouda dtypes module methods
+"""
+
+SUPPORTED_NP_DTYPES = [
+    bool,
+    int,
+    float,
+    str,
+    np.bool_,
+    np.int64,
+    np.float64,
+    np.uint8,
+    np.uint64,
+    np.str_,
+]
+
+
+class TestDTypes:
+    @pytest.mark.parametrize("dtype", SUPPORTED_NP_DTYPES)
+    def test_check_np_dtype(self, dtype):
+        dtypes.check_np_dtype(np.dtype(dtype))
+
+    def test_check_np_dtype_errors(self):
+        with pytest.raises(TypeError):
+            dtypes.check_np_dtype(np.dtype(np.int16))
+        with pytest.raises(TypeError):
+            dtypes.check_np_dtype("np.str")
+        with pytest.raises(TypeError):
+            dtypes.check_np_dtype(ak.bigint)
+
+    def test_translate_np_dtype(self):
+        for b in [np.bool_, bool]:
+            assert ("bool", 1) == dtypes.translate_np_dtype(np.dtype(b))
+
+        for s in [np.str_, str]:
+            assert ("str", 0) == dtypes.translate_np_dtype(np.dtype(s))
+
+        assert ("int", 8) == dtypes.translate_np_dtype(np.dtype(np.int64))
+        assert ("uint", 8) == dtypes.translate_np_dtype(np.dtype(np.uint64))
+        assert ("float", 8) == dtypes.translate_np_dtype(np.dtype(np.float64))
+        assert ("uint", 1) == dtypes.translate_np_dtype(np.dtype(np.uint8))
+
+    def test_resolve_scalar_dtype(self):
+        for b in True, False:
+            assert "bool" == dtypes.resolve_scalar_dtype(b)
+
+        for i in np.iinfo(np.int64).min, -1, 0, 3, np.iinfo(np.int64).max:
+            assert "int64" == dtypes.resolve_scalar_dtype(i)
+
+        floats = [
+            -np.inf,
+            np.finfo(np.float64).min,
+            -3.14,
+            -0.0,
+            0.0,
+            7.0,
+            np.finfo(np.float64).max,
+            np.inf,
+            np.nan,
+        ]
+        for f in floats:
+            assert "float64" == dtypes.resolve_scalar_dtype(f)
+
+        for s in "test", '"', " ", "":
+            assert "str" == dtypes.resolve_scalar_dtype(s)
+        assert "<class 'list'>" == dtypes.resolve_scalar_dtype([1])
+
+        assert "uint64" == dtypes.resolve_scalar_dtype(2 ** 63 + 1)
+        assert "bigint" == dtypes.resolve_scalar_dtype(2 ** 64)
+
+    @pytest.mark.parametrize("size", pytest.prob_size)
+    def test_pdarrays_datatypes(self, size):
```

Comment:
i don't really think we gain anything by parameterizing this on `size`. I don't think having this be large will have anything to do with the dtype

Reply:
Removed